### PR TITLE
Lovelace alignments for all cards

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

More alignment of the lovelace pages.

Main changes:
1. Put types into code tags.
2. Consistent description for the `theme` config property
3. Add full stop at end of description.
4. Resolution of markdown warnings.
5. Consistent wording / spelling.
6. Consistent heading types.

Note: Does not include `entities` page since already covered in https://github.com/home-assistant/home-assistant.io/pull/16214.

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/trigger.markdown [hunk 1]
```diff
@@ -435,7 +435,7 @@ automation:
 
 #### Input Datetime
 
-The Entity ID of an [Input Datetime](/integrations/input_datetime/).
+The Entity ID of an [input datetime](/integrations/input_datetime/).
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]: This refers to a name of integration and thus should be capitalized here.

File: source/_lovelace/alarm-panel.markdown [hunk 2]
```diff
@@ -23,22 +23,22 @@ entity: alarm_control_panel.alarm
 {% configuration %}
 type:
   required: true
-  description: alarm-panel
+  description: "`alarm-panel`"
   type: string
 entity:
   required: true
-  description: "Entity ID of `alarm_control_panel` domain"
+  description: "Entity ID from `alarm_control_panel` domain."
```
Comment (frenck) [inline source/_lovelace/alarm-panel.markdown]: Incorrect. `of` was the right naming here. It is a type of. From would have been correct if provided by another integration, but that is not the context in this case.

File: source/_lovelace/alarm-panel.markdown [hunk 3]
```diff
@@ -23,22 +23,22 @@ entity: alarm_control_panel.alarm
...
   type: string
 name:
   required: false
-  description: Overwrites Friendly Name
+  description: Overwrites friendly name.
   type: string
-  default: Current State of Alarm Entity
+  default: Current state of alarm entity.
```
Comment (frenck) [inline source/_lovelace/alarm-panel.markdown]:
```suggestion
  default: Current state of the alarm entity.
```

File: source/_lovelace/alarm-panel.markdown [hunk 4]
```diff
@@ -50,13 +50,13 @@ states:
       description: Arm Custom Bypass
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/alarm-panel.markdown]: Don't use full links for internal links.

```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/calendar.markdown [hunk 5]
```diff
@@ -23,7 +23,7 @@ Initial View:
 Entities:
   description: The calendar entities that will be displayed in the card.
 Theme:
-  description: Theme your card using any installed theme in your HA environment.
+  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/calendar.markdown [hunk 6]
```diff
@@ -49,7 +49,7 @@ entities:
   type: list
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/entity.markdown [hunk 7]
```diff
@@ -11,38 +11,38 @@ The Entity card gives you a quick overview of your entity's state.
...
   required: false
-  description: Unit of Measurement given to data
+  description: Unit of measurement given to data.
   type: string
-  default: "Unit Of Measurement given by entity"
+  default: Unit of measurement given by entity.
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/entity.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/gauge.markdown [hunk 8]
```diff
@@ -23,58 +23,58 @@ entity: sensor.cpu_usage
...
   type: string
 entity:
   required: true
-  description: "Entity id to show"
+  description: Entity ID to show.
   type: string
 name:
   required: false
-  description: Name of Gauge Entity
+  description: Name of gauge entity entity.
```
Comment (frenck) [inline source/_lovelace/gauge.markdown]:
```suggestion
  description: Name of the gauge entity.
```

File: source/_lovelace/gauge.markdown [hunk 9]
```diff
@@ -23,58 +23,58 @@ entity: sensor.cpu_usage
...
   required: false
-  description: Unit of Measurement given to data
+  description: Unit of measurement given to data.
   type: string
-  default: "Unit Of Measurement given by entity"
+  default: Unit of measurement given by entity.
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/gauge.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/glance.markdown [hunk 10]
```diff
@@ -38,20 +38,20 @@ show_icon:
...
 show_state:
   required: false
-  description: Show entity state-text.
+  description: Show entity state text.
   type: boolean
   default: "true"
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/glance.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/humidifier.markdown [hunk 11]
```diff
@@ -11,25 +11,25 @@ The Humidifier card lets you control and monitor humidifiers, dehumidifiers, and
...
   required: false
-  description: Name of Entity
+  description: Name of entity.
   type: string
-  default: Entity Name
+  default: Entity name
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/humidifier.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/light.markdown [hunk 12]
```diff
@@ -23,25 +23,25 @@ entity: light.bedroom
...
 icon:
   required: false
   description: Overwrites icon.
   type: string
-  default: Entity Domain Icon
+  default: Entity domain icon
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`.
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/light.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/logbook.markdown [hunk 13]
```diff
@@ -24,7 +23,7 @@ Title:
 Hours to show:
   description: The number of hours in the past that will be tracked in the card.
 Theme:
-  description: Theme your card using any installed theme in your HA environment.
+  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/logbook.markdown]:
```suggestion
  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/logbook.markdown [hunk 14]
```diff
@@ -51,7 +50,7 @@ hours_to_show:
   default: 24
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`."
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/logbook.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/markdown.markdown [hunk 15]
```diff
@@ -41,7 +41,7 @@ entity_id:
   description: "A list of entity IDs so a template in `content:` only reacts to the state changes of these entities. This can be used if the automatic analysis fails to find all relevant entities."
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/markdown.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/picture-elements.markdown [hunk 16]
```diff
@@ -33,19 +33,19 @@ camera_view:
...
+  description: Card title.
   type: string
 state_filter:
   required: false
   description: '[State-based CSS filters](#how-to-use-state_filter)'
   type: map
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/picture-elements.markdown [hunk 17]
```diff
@@ -131,42 +131,42 @@ title:
...
     action:
       required: true
-      description: "Action to perform (`more-info`, `toggle`, `call-service`, `navigate`, `url`, `none`)"
+      description: "Action to perform (`more-info`, `toggle`, `call-service`, `navigate`, `url`, `none`)."
       type: string
       default: "`more-info`"
     navigation_path:
       required: false
-      description: "Path to navigate to (e.g.,  `/lovelace/0/`) when `action` defined as `navigate`"
+      description: "Path to navigate to (e.g.,  `/lovelace/0/`) when `action` defined as `navigate`."
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
      description: "Path to navigate to (e.g., `/lovelace/0/`) when `action` defined as `navigate`."
```

File: source/_lovelace/picture-elements.markdown [hunk 18]
```diff
@@ -131,42 +131,42 @@ title:
...
       type: string
       default: none
     url_path:
       required: false
-      description: "Path to navigate to (e.g.,  `https://www.home-assistant.io`) when `action` defined as `url`"
+      description: "Path to navigate to (e.g.,  `https://www.home-assistant.io`) when `action` defined as `url`."
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
      description: "Path to navigate to (e.g., `https://www.home-assistant.io`) when `action` defined as `url`."
```

File: source/_lovelace/picture-elements.markdown [hunk 19]
```diff
@@ -131,42 +131,42 @@ title:
...
       type: string
       default: none
     service:
       required: false
-      description: "Service to call (e.g.,  `media_player.media_play_pause`) when `action` defined as `call-service`"
+      description: "Service to call (e.g.,  `media_player.media_play_pause`) when `action` defined as `call-service`."
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
      description: "Service to call (e.g., `media_player.media_play_pause`) when `action` defined as `call-service`."
```

File: source/_lovelace/picture-elements.markdown [hunk 20]
```diff
@@ -131,42 +131,42 @@ title:
...
       type: string
       default: none
     service_data:
       required: false
-      description: "Service data to include (e.g.,  `entity_id: media_player.bedroom`) when `action` defined as `call-service`"
+      description: "Service data to include (e.g.,  `entity_id: media_player.bedroom`) when `action` defined as `call-service`."
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
      description: "Service data to include (e.g., `entity_id: media_player.bedroom`) when `action` defined as `call-service`."
```

File: source/_lovelace/picture-entity.markdown [hunk 21]
```diff
@@ -61,7 +61,7 @@ show_state:
   default: true
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/picture-glance.markdown [hunk 22]
```diff
@@ -60,12 +60,12 @@ entity:
...
 show_state:
   required: false
-  description: Show entity state-text.
+  description: Show entity state text.
   type: boolean
   default: true
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/picture.markdown [hunk 23]
```diff
@@ -14,15 +14,15 @@ Screenshot of the picture card.
 {% configuration %}
 type:
   required: true
-  description: picture
+  description: "`picture`"
   type: string
 image:
   required: true
-  description: The URL of an image. When you want to store images in your Home Assistant installation use the [hosting files documentation](/integrations/http/#hosting-files). After storing your files, use the /local path, for example /local/filename.jpg .
+  description: "The URL of an image. When you want to store images in your Home Assistant installation use the [hosting files documentation](/integrations/http/#hosting-files). After storing your files, use the `/local` path, for example `/local/filename.jpg`."
```
Comment (frenck) [inline source/_lovelace/picture.markdown]:
```suggestion
  description: "The URL of an image. When you want to store images in your Home Assistant installation use the [hosting files documentation](/integrations/http/#hosting-files). After storing your files, use the `/local` path, for example, `/local/filename.jpg`."
```

File: source/_lovelace/picture.markdown [hunk 24]
```diff
@@ -14,15 +14,15 @@ Screenshot of the picture card.
...
   type: string
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/picture.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/plant-status.markdown [hunk 25]
```diff
@@ -20,16 +20,16 @@ type:
...
   required: false
-  description: Overwrites friendly name
+  description: Overwrites friendly name.
   type: string
-  default: Entity Name
+  default: Entity name
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/plant-status.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/sensor.markdown [hunk 26]
```diff
@@ -11,46 +11,46 @@ The Sensor card gives you a quick overview of your sensors state with an optiona
...
 hours_to_show:
   required: false
-  description: Hours to show in graph
+  description: Hours to show in graph.
   type: integer
   default: 24
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/sensor.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/shopping-list.markdown [hunk 27]
```diff
@@ -25,15 +24,15 @@ type: shopping-list
...
   type: string
 title:
   required: false
-  description: Title of Shopping List
+  description: Title of shopping list.
   type: string
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/shopping-list.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/thermostat.markdown [hunk 28]
```diff
@@ -11,25 +11,25 @@ The Thermostat card gives control of your [climate](/integrations/#climate) enti
...
 name:
   required: false
   description: Overwrites friendly name.
   type: string
-  default: Name of Entity.
+  default: Name of entity.
 theme:
   required: false
-  description: Set to any theme within `themes.yaml`
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/thermostat.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/weather-forecast.markdown [hunk 29]
```diff
@@ -26,7 +25,7 @@ Show Forecast:
 Secondary Info Attribute:
   description: Here you can specify a secondary attribute to show under the current temperature. Ex. Extrema, Precipitation, Humidity. If not set, it will default to Extrema (High/Low) if available, if not available then Precipitation and if precipitation isn't available then Humidity.
 Theme:
-  description: Theme your card using any installed theme in your HA environment.
+  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/weather-forecast.markdown]:
```suggestion
  description: Name of any loaded theme to be used for this card. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

File: source/_lovelace/weather-forecast.markdown [hunk 30]
```diff
@@ -68,7 +66,7 @@ secondary_info_attribute:
   default: Defaults to `extrema` if available, if not available then `precipitation` and if precipitation isn't available then `humidity`.
 theme:
   required: false
-  description: "Set to any theme within `themes.yaml`"
+  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](https://www.home-assistant.io/integrations/frontend/).
```
Comment (frenck) [inline source/_lovelace/weather-forecast.markdown]:
```suggestion
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/automation/trigger.markdown (hunk 1 above)
Comment (spacegaier) [reply source/_docs/automation/trigger.markdown]: I think I saw others that were lower case, which is why I adjusted here. Will revert.

When searching for `](/integrations/` in the docs, there are many instances.

**Review by frenck — APPROVED** [review]
Thanks, @spacegaier 👍

**Pull request merged**